webapp/channels/src/components/update_user_group_modal/__snapshots__/update_user_group_modal.test.tsx.snap

```diff
@@ -110,13 +110,13 @@ exports[`component/update_user_group_modal should match snapshot 1`] = `
         className="update-buttons-wrapper"
       >
         <button
-          className="btn update-group-back"
```

Comment:
Maybe I missed this one, but you probably can remove this from `update_user_group_modal.scss` since this class is (apparently) no longer used.